libs/judicial-system/types/src/lib/notification.ts

```diff
@@ -64,6 +66,8 @@ export enum NotificationType {
   SERVICE_FAILED = SubpoenaNotificationType.SERVICE_FAILED,
   SPOKESPERSON_ASSIGNED = CivilClaimantNotificationType.SPOKESPERSON_ASSIGNED,
   INDICTMENTS_WAITING_FOR_CONFIRMATION = InstitutionNotificationType.INDICTMENTS_WAITING_FOR_CONFIRMATION,
+  INDICTMENT_SENT_TO_PRISON_ADMIN = DefendantNotificationType.INDICTMENT_SENT_TO_PRISON_ADMIN,
```

Comment:
For easy reading please consider grouping these with other defendant notifications.